charts/jenkins/pod-templates/nuxeo-web-ui-ftests.yaml.gotmpl

```
@@ -0,0 +1,33 @@
+- id: "nuxeo-web-ui-ftests"
+  name: "nuxeo-web-ui-ftests"
+  label: "nuxeo-web-ui-ftests"
+  imagePullSecrets:
+  - name: {{ .Values.podTemplates.imagePullSecretName }}
+  serviceAccount: {{ .Values.podTemplates.serviceAccount }}
+  nodeUsageMode:  {{ .Values.podTemplates.nodeUsageMode }}
+  containers:
+  {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
+  - args: ""
+    command: ""
+    image: "docker.platform.dev.nuxeo.com/nuxeo/builder-java11-chrome-nodejs14:latest"
```

```suggestion
    image: "{{ .Values.docker.registry }/nuxeo/builder-java11-chrome-nodejs14:latest"
```
for dynamic replacement depending on the environment: staging/production